src/services/share-buttons.service.ts

```diff
@@ -1,39 +1,18 @@
-import { Injectable } from '@angular/core';
-import { Http, Jsonp, Headers, RequestOptions } from '@angular/http';
-import { Observable } from 'rxjs/Observable';
-
 import 'rxjs/add/operator/map';
 import 'rxjs/add/observable/empty';
 import 'rxjs/add/operator/catch';
 
-import { ShareArgs } from '../helpers/share-buttons.class';
-import { ShareProvider } from '../helpers/share-provider.enum';
-import { ShareButtonsInterface } from './share-buttons.interface';
-import { ShareLinks } from './share-links.functions';
-
-
-
-/** Prepare gPlus count request body   */
-export const gplusCountBody = (url) => {
-    return [{
-        method: 'pos.plusones.get',
-        id: 'p',
-        params: {
-            nolog: true,
-            id: url,
-            source: 'widget',
-            userId: '@viewer',
-            groupId: '@self'
-        },
-        jsonrpc: '2.0',
-        key: 'p',
-        apiVersion: 'v1'
-    }];
-};
+import { Injectable } from '@angular/core';
+import { Http, Jsonp, Headers, RequestOptions } from '@angular/http';
+import { Observable } from 'rxjs/Observable';
 
+import { ShareArgs } from '../classes/share-buttons.class';
+import { ShareProvider } from '../helpers/share-provider.enum';
+import { ShareLinks } from '../helpers/share-links.functions';
+import { Helper } from '../helpers/share.helper';
 
 @Injectable()
-export class ShareButtonsService implements ShareButtonsInterface {
+export class ShareButtonsService {
```

Comment:
Unrelated to this commit! should be done in separate commit (eg: 'refactoring of ShareButtonsService')